ca.vlacroix.Tally.json

```diff
@@ -0,0 +1,50 @@
+{
+	"app-id": "ca.vlacroix.Tally",
+	"runtime": "org.gnome.Platform",
+	"runtime-version": "47",
+	"sdk": "org.gnome.Sdk",
+	"command": "tally",
+	"finish-args": [
+		"--device=dri",
+		"--share=ipc",
+		"--socket=fallback-x11",
+		"--socket=wayland"
+	],
+	"modules": [{
+		"name": "lua",
+		"buildsystem": "simple",
+		"build-commands": [
+			"make",
+			"make install INSTALL_TOP=/app"
+		],
+		"sources": [{
+			"type": "archive",
+			"url": "https://lua.org/ftp/lua-5.4.7.tar.gz",
+			"sha256": "9fbf5e28ef86c69858f6d3d34eccc32e911c1a28b4120ff3e84aaa70cfbf1e30"
+		}]
+	}, {
+		"name": "lgi",
+		"buildsystem": "simple",
+		"build-commands": [
+			"make -C lgi LUA_VERSION=5.4",
+			"make -C lgi install LUA_VERSION=5.4 PREFIX=/app"
+		],
```

Comment:
Use `no-autogen: true`, `make-args` and `make-install-args` and drop `buildsystem: simple`.

Reply:
TIL. Thank you!